CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-# unreleased
+# 5.0.0
```

Comment:
Please leave `# unreleased` section header at the top of this document.

Reply:
It's back where it was now.